extensions/resteasy-reactive/quarkus-resteasy-reactive/kotlin/pom.xml

```diff
@@ -30,7 +30,6 @@
         <dependency>
             <groupId>org.jetbrains.kotlinx</groupId>
             <artifactId>kotlinx-coroutines-core-jvm</artifactId>
-            <optional>true</optional>
```

Comment:
This has to be optional so that it doesn't show up in downstream modules/projects unless specifically asked for.

Reply:
You are correct. I assumed the kotlin module would be imported explicitelly, but it makes integral part of the resteasy reactive.
I'll change it to produce the coroutine scope only if the coroutine dependency is present.
I'll also try to add a compile-time error for when the user creates suspending endpoints, but doesn't have the dependency in the POM. Currently, the request just hangs.

Reply:
Fixed on this [commit](https://github.com/quarkusio/quarkus/pull/16931/commits/1271ce3d9cdc0a4336e6b998429deced0d92e92f)